visidata/guide.py

```diff
@@ -1,27 +1,36 @@
 '''
 # A Guide to VisiData Guides
-Each guide shows you how to use a particular feature in VisiData.
+Each guide shows you how to use a particular feature in VisiData. Gray guides have not been written yet. Why not contribute one?
```

Comment:
Link to some contribution document, otherwise it's a taunt.  Maybe a different exhortation than "Why not?" because it's a non-trivial effort to make one.